spinn_front_end_common/interface/abstract_spinnaker_base.py

```diff
@@ -3602,10 +3642,12 @@ def stop(self, turn_off_machine=None,  # pylint: disable=arguments-differ
 
         if (self._has_ran and self._current_run_timesteps is None and
                 not self._use_virtual_board and not self._run_until_complete):
-            self._do_stop_workflow()
-            # TODO recover from errors?
-            # self._recover_from_error(
-            # e, exc_info[2], executor.get_item("ExecutableTargets"))
+            try:
+                self._do_stop_workflow()
+            except Exception as e:
+                exn = e
+                exc_info = sys.exc_info()
+                self._recover_from_error(e, exc_info[2])
```

Comment:
This is different from the other call to this function; I think it is enough to just pass the exc_info.

Reply:
yes I saw that too.  Release 6.0.0 had it that way

Reply:
I'm not sure how critical it is, but it should probably be consistent.  This seems to just go in to logger.error, and the documentation of that seems to say it should just be sys.exec_info()...